README.md

```diff
@@ -1,56 +1,87 @@
-# Prometheus Operator
+# Prometheus Operator with Prometheus Configurer
 
 ## Description
 
-The Prometheus Operator provides a cluster monitoring solution using
-[Prometheus](https://prometheus.io), which is an open source
-monitoring system and alerting toolkit.
-
-This repository contains a [Juju](https://juju.is/) Charm for
-deploying the monitoring component of Prometheus in a Kubernetes
-cluster. An [alerting service](https://charmhub.io/alertmanager-k8s)
+The Prometheus Operator provides a cluster monitoring solution using 
+[Prometheus](https://prometheus.io), which is an open source monitoring system and alerting 
+toolkit.
+
+This repository is a fork of the
+[prometheus-k8s-operator](https://github.com/canonical/prometheus-k8s-operator).
+It contains a [Juju](https://juju.is/) Charm for deploying the monitoring component 
```

Comment:
Shouldn't we keep the "charm" name consistent?
```suggestion
It contains a [Juju](https://juju.is/) charm for deploying the monitoring component 
```

Reply:
Changed.